src/consumer/consumer.js

```diff
@@ -339,6 +339,7 @@ export default class Consumer {
    * write them only once.
    */
   async writeToComponentsDir({
+    silentClientResult,
```

Comment:
to me, it's not clear the differences between 'silentClientResults' and 'verbose'. I suggest adding a comment explaining what they do.